spec/v3/services/subscriptions/create_spec.rb

```diff
@@ -91,7 +98,16 @@
           '@representation' => 'standard',
           'id' => 1234,
           'valid_to' => '2017-11-28T00:09:59Z',
-          'plan' => 'travis-ci-ten-builds',
+          'plan' => {
+            '@type' => 'plan',
+            '@representation' => 'minimal',
```

Comment:
Is it possible to make this `standard` instead? It looks to me that it already has all the attributes anyway.

Reply:
It does indeed, apparently the v3 framework makes it minimal when you embed it in another resource, I'll have a look to see if it's possible to force it to standard.